src/components/previewCard/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+import UploadImagesIcon from "../icons/uploadImage"
+import ArrowRightTeal from "../icons/arrowRight"
+import { FixedCard } from "../card/fixed"
+
+interface Props {
+  name: string
+  location: {
+    address?: string
+    zipCode: string
+    city?: string
+  }
+  title?: string
+  previewLabel: string
+  image?: string
+  logo?: string
+  link?: string
+  editMode?: boolean
+  handleOnClick?: () => void
+  handleOnError?: (e) => void
+  handleOnClickText?: () => void
```

Comment:
for my understanding - when do `handleOnClick` and `handleOnClickText` differ?

Reply:
These are used on Listings for gtmHelper : the label for the first one is "click image" and the label for the second one is "click text" 😓

Reply:
I would vote for simplifying this. pretty sure we don't care about it on such a level of detail 🤔